### PR TITLE
Removed the landing page type exclusion

### PR DESCRIPTION
**Pull request by jmiraNR**
## Give us some context

* Landing pages remain and need to be translated. Updating the exclusion file to allow for translation.

### CODE REVIEW

File: scripts/actions/utils/docs-content-tools/i18n-exclusions.yml [hunk 1]
```diff
@@ -13,4 +13,3 @@ excludePath:
     - src/content/docs/infrastructure/manage-your-data/data-instrumentation/host-integrations-metrics.mdx
 excludeType:
   ja-JP:
```
Comment (roadlittledawn) [inline scripts/actions/utils/docs-content-tools/i18n-exclusions.yml:15]: if empty i believe you can just remove the whole shebang

**Review by roadlittledawn — CHANGES_REQUESTED** [review]

File: scripts/actions/utils/docs-content-tools/i18n-exclusions.yml (hunk 1 above)
Comment (jmiraNR) [reply scripts/actions/utils/docs-content-tools/i18n-exclusions.yml:15]: I'd rather keep it so I have the template in case we need to exclude types.

Comment (roadlittledawn) [reply scripts/actions/utils/docs-content-tools/i18n-exclusions.yml:15]: since code runs on this, we should really remove this if not used.

Comment (roadlittledawn) [reply scripts/actions/utils/docs-content-tools/i18n-exclusions.yml:15]: we can add some doc on this so it's clear what options are available and how to use it.

**Review by roadlittledawn — APPROVED** [review]
TY! :shipit:

**Pull request merged**